avatar.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <title>Avatar</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="sidebar">
+
+
+            <header>
+                My Components!
+            </header>
+            <ul>
+                <li><a href="avatar.html">AVATAR</a></li>
+                <li><a href="alert.html">ALERT</a></li>
+                <li><a href="#">BADGE</a></li>
+                <li><a href="#">BUTTON</a></li>
+                <li><a href="#">CARD</a></li>
+                <li><a href="#">IMAGE</a></li>
+                <li><a href="#">INPUT</a></li>
+
+
+            </ul>
+
+        </div>
+        <header class="head">
+            AVATAR
+        </header>
+
+
+        <div class="container">
+            <div class="i-container">
+
+                <div class="center-container">
+                    <h3 class="headers">Round Avatar!</h3>
+                    <div>
+                        <span> <img class="XXL" src="garv.png"></span>
+
+                        <span> <img class="XL" src="garv.png"></span>
+                        <span> <img class="L" src="garv.png"></span>
+                        <span> <img class="M" src="garv.png"></span>
+                        <span> <img class="S" src="garv.png"></span>
```

Comment:
Try using the same image as square ones and change the look through border radius

Reply:
I changed and used same image both for my round and square avatar